Frontend/src/components/TableEditors/Rentals/Rentals.svelte

```diff
@@ -53,4 +66,6 @@
   {filters}
   database={$rentalDb}
   {rowBackgroundColorFunction}
-  popupFormularComponent={RentalPopupFormular} />
+  {cellBackgroundColorFunction}
```

Comment:
I thought we could maybe just merge these two functions into one, since coloring a row is practically coloring all cells? Maybe we could do one function returning a color for each cell?

Reply:
That would be possible!

Reply:
I'm not sure yet, I think it might be useful to have both, otherwise the cell-style-function becomes quite bloated. What do you think?